quadkey/bq/sql/KRING.sql

```diff
@@ -15,15 +15,16 @@ AS """
     {
         throw new Error('NULL argument passed to UDF');
     }
-
-    let left      = sibling(quadint,'left').toString();
-    let topleft   = sibling(left,'up').toString();
-    let downleft  = sibling(left,'down').toString();
-    let right     = sibling(quadint,'right').toString();
-    let topright  = sibling(right,'up').toString();
-    let downright = sibling(right,'down').toString();
-    let up        = sibling(quadint,'up').toString();
-    let down      = sibling(quadint,'down').toString();
-
-    return [left, topleft, downleft, right, topright, downright, up, down, quadint];
+    if(distance == null)
+    {
+        distance = 0;
```

Comment:
Isn't 0 distance the own `quadint`? So IMO NULL -> 1, which should return the old behaviour.